cardinal/server/handler/health.go

```diff
@@ -11,11 +11,11 @@ type GetHealthResponse struct {
 
 // GetHealth godoc
 //
-//	@Summary		Get information on status of world-engine
-//	@Description	Displays information on http server and world game loop
-//	@Produce		application/json
-//	@Success		200	{object}	GetHealthResponse
-//	@Router			/health [get]
+//	@Summary      Get Cardinal's health status
+//	@Description  Retrieves the running status of the Cardinal server and game loop
```

Comment:
Should just be `Retrieves the status of the server and game loop` for both summary and description